src/sentry/search/events/fields.py

```diff
@@ -2359,6 +2408,9 @@ def resolve_function(self, function: str, match: Optional[Match[str]] = None) ->
 
         name, arguments, alias = self.parse_function(match)
         snql_function = self.function_converter.get(name)
+
+        arguments = snql_function.format_as_arguments(name, arguments, self.params)
```

Comment:
This works for now because we're only working with a simple subset of `CountColumn` and numbers. But this method normalizes more complex arguments into the JSON syntax and will fail as we continue with the migration.

Reply:
Yeah you are right, it's going to get tricky and I kinda started running into it already... going to start a thread in Slack about this